src/redux/categories/categories.js

```diff
@@ -0,0 +1,7 @@
+const categories = () => (
+  <div>
+    <p>Under construction</p>
+  </div>
+);
+
+export default categories;
```

Comment:
- [ ] You created your Category component under the `redux` directory. All components should live under the `components` directory. This `redux` directory is solely meant for redux configurations. Kindly extract this component on to the **components** folder

Reply:
![image](https://user-images.githubusercontent.com/50904584/145826882-7ae9ba90-5ff6-4b3b-8aaf-8a8250a1417c.png)

Print screen from the task description.
Could you please provide more info?

Reply:
Hey @aakbarkhan as you can see there are two folders (**components** and **redux**). The components folder should contain all your components. And the redux folder should contain redux-related configurations (like actions, action types reducers, and so on). What you have in **redux**/categories/categories is a component, and so it should be under the components folder.

Reply:
Thank you.